cwms-data-api/src/main/java/cwms/cda/data/dao/location/kind/OutletDao.java

```diff
@@ -223,18 +207,25 @@ private CompoundOutletRecord mapToCompoundRecord(List<String> table, String offi
         if (!table.isEmpty()) {
             outlet = table.get(0);
         }
-        return new CompoundOutletRecord.Builder().withOutletId(
+        return new VirtualOutletRecord.Builder().withOutletId(
```

Comment:
Missing projectId

Reply:
The project id is part of the VirtualOutlet, not the records.

Reply:
ah gotcha